orchagent/muxorch.cpp

```diff
@@ -1353,6 +1355,12 @@ bool MuxOrch::isNeighborActive(const IpAddress& nbr, const MacAddress& mac, stri
         return ptr->isActive();
     }
 
+    if (alias == nullptr || alias == ""){
+        SWSS_LOG_ERROR("Interface alias is empty", alias.c_str());
```

Comment:
Change to WARN level. In any case, this is not the place its having an issue on exception